ancestry_extract.py

```diff
@@ -627,8 +724,13 @@ def user_media(session, line):
             if 'image' in metadata:
                 if not os.path.isfile(metadata['image']):
                     process_data = True
-        except Exception:
+
+            if check_url_note(url_note, metadata):
```

Comment:
This should flag it to download the new clipping as a pdf, even if the media (image) was already downloaded before. But it won't download the clipping a second time.